swarm/api/ws/server.go

```diff
@@ -0,0 +1,54 @@
+package ws
```

Comment:
so can we just use the code in cmd/geth to allow the same way of starting a WS server?
No need for this special treatment right?